ppx/ppx_cstruct.ml

```diff
@@ -526,15 +526,20 @@ let constr_enum = function
   | {pcd_loc = loc; _} ->
     loc_err loc "invalid cenum variant"
 
+let get_len = function
+  | [ ({txt = "len"; _},
+       PStr
+         [{pstr_desc =
+             Pstr_eval ({pexp_desc = Pexp_constant (Pconst_integer (sz, None)); _}, _)
+          ; _}])] ->
+    Some (int_of_string sz)
```

Comment:
It's probably worth checking that it's a positive, non-zero integer here.

Reply:
sure, done!